manifests/tftp.pp

```diff
@@ -29,4 +29,75 @@
       root => $foreman_proxy::tftp_root,
     }
   }
+
+  case $::osfamily {
```

Comment:
But what to do in there? Every single OS deals with Grub differently. Not sure what I should put in.